packages/dto/src/tables/dynamo/users/profile.rs

```diff
@@ -0,0 +1,105 @@
+use crate::{Error, Result};
+use aws_sdk_dynamodb::types::AttributeValue;
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct DynamoProfile {
+    // DynamoDB Keys
+    pub pk: String, // USER#{user_id}
+    pub sk: String, // PROFILE
```

Comment:
```suggestion
    pub sk: String, // PROFILE
```

Make enum to share among all data type in an single table.